raiden/tests/unit/transfer/mediated_transfer/test_mediation_fee.py

```diff
@@ -326,16 +340,34 @@ def test_fee_round_trip(flat_fee, prop_fee, imbalance_fee, amount):
         )
     )
 
+    # How much do we need to send so that the target receives `amount`? PFS-like calculation.
     fee_calculation = get_initial_payment_for_final_target_amount(
         final_amount=PaymentAmount(amount), channels=[payer_channel_backwards, payee_channel]
     )
+    assume(fee_calculation)  # There is not enough capacity for the payment in all cases
```

Comment:
Do you see direct improvements with the test cases when adding `assume`s?

Reply:
I can't run the following checks when this assume is not true. But since I use assume, hypothesis will at least make sure that many test cases will have been tried where all assumes are true.